Sloth.Web/Views/Shared/_Layout.cshtml

```diff
@@ -201,6 +213,21 @@
         <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-validation-unobtrusive/3.2.11/jquery.validate.unobtrusive.min.js"></script>
     </environment>
 
+    <script type="text/javascript">
+        $('#team-select').on('change', function (e) {
+            var optionSelected = $("option:selected", this);
+            var valueSelected = this.value;
+            if(valueSelected !== "")
+            {
+                window.location = "\\" + valueSelected + "\\transactions";
```

Comment:
Why backslashes? URLs would normally be forward

Reply:
![image](https://user-images.githubusercontent.com/469010/185204936-15548c78-96a0-468a-8a3c-e2bc3b86f8f5.png)